src/main/java/svc/data/municipal/MunicipalityDAO.java

```diff
@@ -81,12 +89,13 @@ public void processRow(ResultSet rs) {
                     municipality.id = new HashableEntity<Municipality>(Municipality.class,municipalityId);
                     municipality.name = rs.getString(MUNICIPALITY_NAME_COLUMN_NAMER);
                     municipality.courts = Lists.newArrayList(courtId);
+                    municipality.isSupported = (rs.getString("citation_datasource_id") != null)?true:false;
```

Comment:
The ternary here is redundant..Just assign like this:
`municipality.isSupported = rs.getString("citation_datasource_id") != null`